examples/platforms/posix/misc.c

```diff
@@ -43,6 +43,7 @@ extern char **gArguments;
 #endif
 
 static otPlatResetReason sPlatResetReason = OT_PLAT_RESET_REASON_POWER_ON;
+static otPlatPowerState  sPlatPowerState  = OT_PLAT_POWER_STATE_ONLINE;
```

Comment:
Not asking that we fix this now but I find the name OT_PLAT_POWER_STATE_ONLINE to be misleading.  The device is technically not online as that suggests a network connection of some sort.  Perhaps at a later date we should revisit these names?